mmocr/datasets/preparers/parsers/naf_parser.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import json
+from typing import Dict, List, Tuple
+
+import numpy as np
+
+from ..data_preparer import DATA_PARSERS
+from .base import BaseParser
+
+
+@DATA_PARSERS.register_module()
+class NAFTextDetAnnParser(BaseParser):
```

Comment:
`NAFTextDetAnnParser` works for all the tasks, so it can be renamed as `NAFAnnParser`.